tests/unit/alp_views.cpp

```diff
@@ -164,10 +164,15 @@ void alpProgram( const size_t &n, alp::RC &rc ) {
 	print_vector( "Mcol", Mcol );
 	std::cout << "------------" << std::endl;
 
-	// diagonal view
+	// diagonal view on a general (non-square) matrix
 	auto Mdiag = alp::get_view< alp::view::Views::diagonal >( M );
```

Comment:
You don't need to refer to `Views` (or the in general an enum name) to get to `diagonal`. `alp::view` already defines the scope and each enum items are valid constants in that scope.